src/domain/cohort.js

```diff
@@ -12,6 +12,12 @@ export async function createCohort() {
   return new Cohort(createdCohort.id)
 }
 
+export async function findCohorts() {
+  const allCohorts = await dbClient.cohort.findMany({})
```

Comment:
Nice use of the domain layer! Very consistent!

Reply:
In  the user domain, the results from database calls are returned as instances of the domain class, rather than the raw prisma objects - see: https://github.com/boolean-uk/team-dev-server/blob/main/src/domain/user.js#L186

In this case it doesn't change much since the domain object is so simple, but it might be worth that small update just to make them the same?